core/domain/skill_services.py

```diff
@@ -277,6 +280,190 @@ def save_new_skill(committer_id, skill):
         }])
 
 
+def apply_change_list(skill_id, change_list):
+    """Applies a changelist to a skill and returns the result.
+
+    Args:
+        skill_id: str. ID of the given skill.
+        change_list: list(dict). A change list to be applied to the given
```

Comment:
Is it possible to use domain objects here instead?

Reply:
Done.